test/utils/clientset_generated.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
This is back upstream https://github.com/kubernetes-sigs/cluster-api/tree/master/pkg/client/clientset_generated/clientset

Reply:
Do we know pick this up from upstream?

Reply:
We don't. It was not needed at the end.